draft-ietf-mmusic-sdp-bundle-negotiation.xml

```diff
@@ -1119,21 +1114,115 @@ SDP Answer
+      line, the following steps MUST be followed for each BUNDLE
+      group.</t>
+
+      <t>
+        <list>
+
+          <t>Construct a table mapping MID to "m=" line for each
+          "m=" line in this BUNDLE group.</t>
+
+          <t>Construct a table mapping incoming SSRC to "m=" line for
+          each "m=" line in this BUNDLE group and for each SSRC
+          configured for receiving in that "m=" line.</t>
+
+          <t>Construct a table mapping outgoing SSRC to "m=line" for
+          each "m=" line in this BUNDLE group and for each SSRC
+          configured for sending in that "m=" line.</t>
+
+          <t>Construct a table mapping payload type to "m=" line for
+          each "m=" line in the BUNDLE group and for each payload type
+          configured for receiving in that "m=" line.  If any payload
+          type is configured for receiving in more than one "m=" line
+          in the BUNDLE group, do not it include it in the table.</t>
+        </list>
+      </t>
+
+      <t>As "m=" lines are added or removed from the BUNDLE groups, or
+      their configurations are changed, the tables above must also
+      be updated.</t>
+
+      <t>For each RTP packet received, the following steps MUST be
+      followed to route the packet to the correct "m=" section within
+      a BUNDLE group:</t>
+
+      <t>
+        <list>
+          <t>If the packet has a MID and that MID is not in the table
+          mapping MID to "m=" line, drop the packet and stop.</t>
+
+          <t>If the packet has a MID and that MID is in the table
+          mapping MID to "m=" line, update the incoming SSRC mapping
+          table to include an entry that maps the packet's SSRC to the
+          "m=" line for that MID.</t>
+
+          <t>If the packet's SSRC is in the incoming SSRC mapping
+          table, route the packet to the associated "m=" line and
+          stop.</t>
+
+          <t>If the packet's payload type is in the payload type
+          table, update the the incoming SSRC mapping table to include
+          an entry that maps the packet's SSRC to the
+          "m=" line for that payload type. In addition, route the packet
+          to the associated "m=" line and stop.</t>
+
+          <t>Otherwise, drop the packet.</t>
+        </list>
+      </t>
+
+      <t>For each RTCP packet received (including each RTCP packet that is part
+      of a compound RTCP packet), the following type-specific handling
+      MUST be performed to route the packet:</t>
+
+      <t>
+        <list>
+          <t>If the packet is of type SR, and the sender SSRC for the
+          packet is found in the incoming SSRC table, deliver a copy of the
+          packet to the "m=" line associated with that SSRC. In addition,
+          for each report block in the report whose SSRC is found in the
+          outgoing SSRC table, deliver a copy of the RTCP packet to the
+          "m=" line associated with that SSRC.</t>

```

Comment:
Linguistic: an m= line is not an active object. Suggest "to the m= line" => "to the handler designated for the m= line".